includes/course-theme/class-sensei-course-theme-option.php

```diff
@@ -76,6 +76,7 @@ public function init( $sensei ) {
 		add_action( 'template_redirect', [ $this, 'ensure_learning_mode_url_prefix' ] );
 		add_action( 'template_redirect', [ Sensei_Course_Theme_Lesson::instance(), 'init' ] );
 		add_action( 'template_redirect', [ Sensei_Course_Theme_Quiz::instance(), 'init' ] );
+		add_filter( 'show_admin_bar', [ $this, 'hide_admin_bar_for_non_editors' ] );
```

Comment:
Need to change this to the new name :)

```suggestion
		add_filter( 'show_admin_bar', [ $this, 'hide_admin_bar_for_non_editors' ] );
```
```suggestion
		add_filter( 'show_admin_bar', [ $this, 'show_admin_bar_only_for_editors' ] );
```

Reply:
Oops!